### DISCUSSION TITLE
"hydra serve all" won't work with the memory backend?

### DISCUSSION THREAD

**Discussion by rawnsley** [ory/hydra#2959] [Q&A] [answered]
According to the [5-minute guide](https://www.ory.sh/hydra/docs/production#exposing-administrative-and-public-api-endpoints), `hydra serve all` is incompatible with the `memory` backend. I presume that means when you set _DSN=memory_ and don't use a separate database.

I can't find any other reference to this (I haven't looked for long to be fair), but it seems like a big gotcha not to mention in the documentation for `hydra serve` given that for most people starting with hydra, `memory` seems like a simpler option than a database you need to maintain and migrate.

**Comment by aeneasr** [comment, marked as answer]
Yes that looks incorrect! `hydra serve all` is actually compatible with memory mode, where as running the process separated is not!

@vinckr can we update the docs here?
